lib/utils/ember.js

```diff
@@ -299,8 +299,20 @@ function isEmberProxy(context, node) {
   return isEmberArrayProxy(context, node) || isEmberObjectProxy(context, node);
 }
 
-function isInjectedServiceProp(node) {
-  return isPropOfType(node, 'service') || isPropOfType(node, 'inject');
+function isInjectedServiceProp(node, importedEmberName, importedInjectName) {
+  return (
```

Comment:
Can you add bullet-point list in a comment inside this function showing what forms this looks for?

```
* service()
* Ember.inject.service()
```